### PR TITLE
closes #2

### LINKED ISSUE #2: Translate README into English
Hi, I would like to translate the README into English (and your parameters/documentation) so more people can get a chance to play around with the API. Let me know if you're interested!

### PR DESCRIPTION
**Pull request by adybas**
Hi, I went ahead and wrote out an English readme with documentation. I really liked this API and just want to make sure the documentation can reflect how awesome it is.

### CODE REVIEW

**Comment by segeda** [comment]
@adybas Great work! 👍 
But maybe update original README.md with your english version and delete README.en-US.md. Every czech developer can read english documentation.
Thanks

**Comment by adybas** [comment]
Okay, awesome! I'll go ahead and make those changes; I didn't want to assume that every Czech developer could read them, so that is good to know.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,46 @@
 svatky-api-nodejs
 =================
 
-Svátky API Vám řekne kdo má kdy svátek, nebo kdy má kdo svátek.
\ No newline at end of file
+[Svátky API](https://svatky.adresa.info/) Vám řekne kdo má kdy svátek, nebo kdy má kdo svátek.
+
+[Svatky API](https://svatky.adresa.info/) - Find out which date in the year is your Name Day or who's Name Day is on a certain date.
+
+This API only accpets **GET** requests with three parameters: date, name, lang.
```
Comment (segeda) [inline README.md]:
```suggestion
This API only accept **GET** requests with three parameters: date, name, lang.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,46 @@
...
+* lang -- Formatted to accept either cs or sk
+    * cs || sk
+
+* date -- Formatted date first followed by month. Ex: May 15 => 1505
+    * DDMM
+
+* name -- Formatted for first names only. Ex: Michal, Anna, Karina
+
+
+For example, if we wanted to seach for when the Name Day is of Anna in JSON, we would set the parameters up like so:
```
Comment (segeda) [inline README.md]:
```suggestion
For example, if we wanted to search for when the Name Day is of Anna in JSON, we would set the parameters up like so:
```

**Review by segeda — CHANGES_REQUESTED** [review]
some typos

**Review by adybas — COMMENTED** [review]
Thank you for catching the typos! Fixed them in update. 

Let me know if there is anything else you would like me to add or expand upon in the documentation.

**Pull request merged**